types/underscore/index.d.ts

```diff
@@ -1038,19 +1025,19 @@ declare module _ {
             context?: any): number;
 
         /**
-        * Uses a binary search to determine the index at which the value should be inserted into the list in order
-        * to maintain the list's sorted order. If an iterator is passed, it will be used to compute the sort ranking
-        * of each value, including the value you pass.
-        * @param list The sorted list.
-        * @param value The value to determine its index within `list`.
-        * @param iterator Iterator to compute the sort ranking of each value, optional.
-        * @param context `this` object in `iterator`, optional.
-        * @return The index where `value` should be inserted into `list`.
-        **/
-        sortedIndex<T, TSort>(
-            list: _.List<T>,
-            value: T,
-            iterator?: ((x: T) => TSort) | string,
+         * Uses a binary search to determine the index at which the value should be inserted into `list` in order
```

Comment:
Specifically, it determines the *lowest* index at which the value could be inserted without disturbing the order. This is also known as the lower bound. `sortedLastIndex`, which isn't part of the public interface yet (but will be soonish), would determine the upper bound.

Reply:
I've updated the summary comments to reflect that, thanks!